hw/ip/prim/util/gen-lfsr-seed.py

```diff
@@ -125,15 +125,15 @@ def main():
 localparam int {}LfsrWidth = {};
 typedef logic [{}LfsrWidth-1:0] {}lfsr_seed_t;
 typedef logic [{}LfsrWidth-1:0][$clog2({}LfsrWidth)-1:0] {}lfsr_perm_t;
-localparam lfsr_seed_t RndCnst{}LfsrSeedDefault = {};
-localparam lfsr_perm_t RndCnst{}LfsrPermDefault =
+localparam {}lfsr_seed_t RndCnst{}LfsrSeedDefault = {};
```

Comment:
Question: I think in 99% of all cases, those snippets will end up in a package file. I thus suggest do convert the type from `localparam` to `parameter` by default. WDYT @msfschaffner ?

Reply:
Oh yeah good point. Let's make these parameters.

Reply:
Cool, I've changed it. Thanks for the quick review!